scripts/stackblitz.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+# Simplified start script for the stack blitz environment, which can't handle multiple watchers
+set -x
+set -e
+
+(cd packages/gestalt-design-tokens && yarn build)
```

Comment:
Notice these are "build" commands, no watcher is enabled to save web memory usage